menuv.lua

```diff
@@ -184,6 +184,40 @@ function MenuV:CreateMenu(title, subtitle, position, r, g, b, size, texture, dic
     return self.Menus[index] or menu
 end
 
+--- Create a menu that inherits properties from another menu
+---@param parent Menu|string Menu or UUID of menu
+---@param overrides table<string, string|number> Properties to override in menu object (ignore parent)
+function MenuV:CreateChildMenu(parent, overrides)
+    overrides = Utilities:Ensure(overrides, {})
+
+    local uuid = Utilities:Typeof(parent) == 'Menu' and parent.UUID or Utilities:Typeof(parent) == 'string' and parent
+
+    if (uuid == nil) then return end
+
+    parentMenu = self:GetMenu(uuid)
+
+    if (parentMenu == nil) then return end
+
+    local menu = CreateMenu({
+        Title = Utilities:Ensure(overrides.title or overrides.Title, parentMenu.Title),
+        Subtitle = Utilities:Ensure(overrides.subtitle or overrides.Subtitle, parentMenu.Subtitle),
+        Position = Utilities:Ensure(overrides.position or overrides.Position, parentMenu.Position),
+        R = Utilities:Ensure(overrides.r or overrides.R, parentMenu.Color.R),
+        G = Utilities:Ensure(overrides.g or overrides.G, parentMenu.Color.G),
+        B = Utilities:Ensure(overrides.b or overrides.B, parentMenu.Color.B),
+        Size = Utilities:Ensure(overrides.size or overrides.Size, parentMenu.Size),
+        Texture = Utilities:Ensure(overrides.texture or overrides.Texture, parentMenu.Texture),
+        Dictionary = Utilities:Ensure(overrides.dictionary or overrides.Dictionary, parentMenu.Dictionary),
+        Namespace = Utilities:Ensure(overrides.namespace or overrides.Namespace, parentMenu.Namespace)
```

Comment:
Namespace can't be override, must always be unique.
https://github.com/ThymonA/menuv/blob/cb675030728cf5998ad202744d865330270c0428/menuv.lua#L372-L374

The namespace is only used to set key binds for a menu, UUID couldn't be used for that because UUID is always random and namespace is always the same.

Reply:
Ahh ok. I wasn't really sure what the namespace did (I didn't really look into it, but now that I think of it, it would've been smart for me to).

Reply:
Namespace is now documented in https://menuv.fivem.io/api/#Menu.Namespace

Thanks for your help and this nice feature ❤️